pkg/cli/server/server.go

```diff
@@ -180,3 +185,11 @@ func knownIPs(ips []string) []string {
 	}
 	return ips
 }
+
+func checkUnixTimestamp() error {
```

Comment:
We want to check that it's before 1971 or something.  For example the system boots and starts at 1970 (unix time 0) and then maybe 2 minutes later k3s starts so the time is not 0, but 2 minutes past 1970 june.  So the easiest is just to pick some old date and make sure it's before that.  Like 1981 :)

Reply:
I see, fixed thanks!